docs/index.rst

```diff
@@ -5,10 +5,6 @@ Robocop documentation
    :maxdepth: 2
    :caption: Contents:
 
-.. include:: ../README.rst
```

Comment:
Now we need to add some text here otherwise the entry page will be empty: 
https://robocop.readthedocs.io/en/bamf-refactor/

Reply:
Yeah, I know.

Reply:
@bhirsz  Added back in https://github.com/MarketSquare/robotframework-robocop/pull/335/commits/d7fa8735b9b0d4b923bd2a1c15f0e573f14d95fb
Can you please check?

Reply:
The url I gave earlier rebuilds with every commit (https://robocop.readthedocs.io/en/bamf-refactor/). It looks good!